instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/tracer/DatabaseClientTracer.java

```diff
@@ -21,10 +20,12 @@
 
   protected static final String DB_QUERY = "DB Query";
 
-  protected final Tracer tracer;
-
   public DatabaseClientTracer() {
-    tracer = GlobalOpenTelemetry.getTracer(getInstrumentationName(), getVersion());
+    super();
```

Comment:
if you accept this, spotless will make you collapse the empty constructor to a single line `public DatabaseClientTracer() {}`
```suggestion
```

Reply:
made updates to placate spotless